src/pybind/mgr/nfs/export.py

```diff
@@ -71,6 +71,17 @@ def normalize_path(path: str) -> str:
     return path
 
 
+def validate_cephfs_path(mgr: 'Module', fs_name: str, path: str) -> None:
```

Comment:
`create_cephfs_export` needs to be fixed to use this helper.

Reply:
'fixed' would not be a good term though :) but yeah this can be used there to save up some lines there

Reply:
> 'fixed' would not be a good term though :) but yeah this can be used there to save up some lines

Not "can be used " but "needs to b used" :)

Reply:
added